src/components/stepforms/widgets/InputText.vue

```diff
@@ -1,6 +1,11 @@
 <template>
   <div class="widget-input-text__container" :class="toggleClassErrorWarning">
-    <label v-if="name" :for="id">{{ name }}</label>
+    <div class="widget-input-text__header">
```

Comment:
`header` is maybe not the best identifier here. It could be below, or aside. Would you be OK with `label` instead?